### DISCUSSION TITLE
RTDose Export

### DISCUSSION THREAD

**Discussion by Fabio-Dan** [e0404/matRad#675] [General]
Apologies if this question has been asked before, I am hoping to export some of my resulting dose grids found at resultGUI.physicalDose as a Dicom RTDose file - is this possible? 

I noticed there seems to be some code labeled as such but can't seem to get my head round as to how to run it? 

Thanks!

**Comment by wahln** [comment]
Check out these lines of code at the end of the first example:
https://github.com/e0404/matRad/blob/fa4769b6ccd96d16c145e746503a8751359b6b4f/examples/matRad_example1_phantom.m#L244-L246

When you run it like this, matRad will automatically export everything that is currently stored in resultGUI.

Alternatively, in case you have named the structures differently (e.g. `resultGUI_someSpecialName`) you can construct the dicom exporter with arguments:
`matRad_DicomExporter(ct,cst,pln,stf,resultGUI_someSpecialName);`

I hope this helps.
